README.md

```diff
@@ -1,95 +1,76 @@
+Supported Android Gradle Plugin Version: `3.0.1`
 
-Key features
------
-
-Augments Java, Kotlin, Groovy bytecode simultaneously!<br />
-Works with background mechanics of jvm-based languages out-of-box!<br />
-[How to teach Android Studio to understand the AspectJ!](IDE)<br />
-May not work properly for AS 3.0 :(
-
-It is easy to isolate your code with aspect classes, that will be simply injected via cross-point functions, named `advices`, into your core application. The main idea is — code less, do more!
-
-AspectJ-Gradle plugin provides supply of all known JVM-based languages, such as Groovy, Kotlin, etc. That means you can easily write cool stuff which may be inject into any JVM language, not only Java itself! :)
-
-To start from you may look at my <a href="https://github.com/Archinamon/AspectJExampleAndroid" target="_blank">example project</a>. And also you may find useful to look at <a href="https://eclipse.org/aspectj/doc/next/quick5.pdf" target="_blank">reference manual</a> of AspectJ language and simple <a href="https://eclipse.org/aspectj/sample-code.html" target="_blank">code snipets</a>. In case aspectj-native not supported by Android Studio (even with IDE-plugin it's using is complicated), you may write a java-classes with aspectj annotations.
-
-Two simple rules you may consider when writing aspect classes.
-- Do not write aspects outside the `src/$flavor/aspectj` source set! These aj-classes will be excluded from java compiler.
-- Do not try to access aspect classes from java/kotlin/etc. In case java compiler doesn't know anything about aspectj, it will lead to compile errors on javac step.
-
-These rules affects only in case you're writing in native aj-syntax.
-You may write aspects in java-annotation style and being free from these limitations.
-
-Usage
------
-
-First add a maven repo link into your `repositories` block of module build file:
-```groovy
-mavenCentral()
-```
-Don't forget to add `mavenCentral()` due to some dependencies inside AspectJ-gradle module.
-
-Add the plugin to your `buildscript`'s `dependencies` section:
-```groovy
-classpath 'com.archinamon:android-gradle-aspectj:3.2.0'
+## Installation
+1. Update your **top-level** `build.gradle` to include the plugin in the build.
++ Update the `buildscript` `repositories` to include jcenter if it is not present 
+and add a `classpath` dependency on the plugin as shown in the following example:
+``` groovy
+buildscript {
+    repositories {
+        jcenter()
+        google()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.0.1'
+        classpath 'com.thunderhead.android:android-gradle-plugin-aspectj:4.0.0'
+    }
+}
 ```
-
-Apply the `aspectj` plugin:
-```groovy
-apply plugin: 'com.archinamon.aspectj'
++ Update the `allprojects` `repositories` closure to include `mavenCentral` as shown in the following example:
+``` groovy
+allprojects {
+    repositories {
+        jcenter()
+        mavenCentral()
```

Comment:
Whatever is available on Maven Central is available on jcenter as well, so do we really need mavenCentral() here?

Reply:
It was explicitly called out in the docs before. I just formalized it. The note was that some mavenCentral dependencies for aspectj were not in jcenter. Not sure that's possible but I kept what was said and just applied it in the cleanup.